lnbits/lnurl.py

```diff
@@ -1,66 +0,0 @@
-from http import HTTPStatus
-from typing import Callable
-
-from fastapi import HTTPException, Request, Response
-from fastapi.responses import JSONResponse
-from fastapi.routing import APIRoute
-from lnurl import LnurlErrorResponse, decode, encode, handle
-from loguru import logger
-
-from lnbits.exceptions import InvoiceError, PaymentError
-
-
-class LnurlErrorResponseHandler(APIRoute):
```

Comment:
forgot to remove this in the initial lnurl lib cleanup, i introduced to responsehandler and i also removed it again from the extensions because just returning a `LnurlErrorResponse` is cleaner